acceptance/internal/materialized_config.go

```diff
@@ -0,0 +1,57 @@
+package internal
+
+import (
+	"encoding/json"
+	"time"
+)
+
+const MaterializedConfigFile = "out.config.json"
+
+type MaterializedConfig struct {
+	GOOS                 map[string]bool     `json:"goos,omitempty"`
+	CloudEnvs            map[string]bool     `json:"cloud_envs,omitempty"`
```

Comment:
Why add aliases CloudEnvs -> cloud_envs? We use CloudEnvs in original config, why materialized should have a different name?

Reply:
Fixed here https://github.com/databricks/cli/pull/3174/commits/29688158ea8a935a687b6253c3d546d3325cbb82